docs/notebooks/image_to_image.ipynb

```diff
@@ -171,7 +171,7 @@
         "but recommended in order to retrieve your run easily and have some context about it.\n",
         "* Furthermore, we had to provide names of the `train_data`.\n",
         "* We set `TripletMarginLoss`.\n",
-        "* Additionally, we use {class}`~finetuner.callback.EvaluationCallback` for evaluation.\n",
+        "* Additionally, we use `finetuner.callback.EvaluationCallback` for evaluation.\n",
```

Comment:
Why is this being changed?

Reply:
It can not be displayed in the notebooks